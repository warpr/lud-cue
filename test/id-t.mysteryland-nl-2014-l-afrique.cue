REM COMMENT l√ªd cue v1
REM MUSICBRAINZ_ALBUM_ID 12d5a9d5-aa95-4f25-aca9-e482b616a3ca
TITLE "Mysteryland NL 2014: L'Afrique"
PERFORMER "ID&T"
FILE "disc01.01.mp3" WAVE
  TRACK 01 AUDIO
    TITLE "L'Afrique"
    PERFORMER "Cincity"
    INDEX 01 00:00:00
FILE "disc01.02.mp3" WAVE
  TRACK 02 AUDIO
    TITLE "L'Afrique"
    PERFORMER "Madison Groove"
    INDEX 01 00:00:00
FILE "disc01.03.mp3" WAVE
  TRACK 03 AUDIO
    TITLE "L'Afrique"
    PERFORMER "Morelasoul"
    INDEX 01 00:00:00
FILE "disc01.04.mp3" WAVE
  TRACK 04 AUDIO
    TITLE "L'Afrique"
    PERFORMER "Soulroots"
    INDEX 01 00:00:00
FILE "disc01.05.mp3" WAVE
  TRACK 05 AUDIO
    TITLE "L'Afrique"
    PERFORMER "R-Jay"
    INDEX 01 00:00:00
FILE "disc01.06.mp3" WAVE
  TRACK 06 AUDIO
    TITLE "L'Afrique"
    PERFORMER "Rico Rosa"
    INDEX 01 00:00:00
FILE "disc01.07.mp3" WAVE
  TRACK 07 AUDIO
    TITLE "L'Afrique"
    PERFORMER "Pascal Morais"
    INDEX 01 00:00:00
FILE "disc01.08.mp3" WAVE
  TRACK 08 AUDIO
    TITLE "L'Afrique"
    PERFORMER "Rancido"
    INDEX 01 00:00:00
