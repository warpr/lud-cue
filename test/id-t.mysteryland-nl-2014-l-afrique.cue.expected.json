{
  "artist": "ID&T",
  "comments": [
    "l√ªd cue v1"
  ],
  "duration": 31105.384488,
  "files": [
    {
      "filename": "disc01.01.mp3",
      "format": "WAVE",
      "tracks": [
        {
          "artist": "Cincity",
          "duration": null,
          "pos": 1,
          "start": {
            "minutes-seconds-frames": "00:00:00",
            "seconds": 0
          },
          "title": "L'Afrique"
        }
      ]
    },
    {
      "filename": "disc01.02.mp3",
      "format": "WAVE",
      "tracks": [
        {
          "artist": "Madison Groove",
          "duration": null,
          "pos": 2,
          "start": {
            "minutes-seconds-frames": "00:00:00",
            "seconds": 0
          },
          "title": "L'Afrique"
        }
      ]
    },
    {
      "filename": "disc01.03.mp3",
      "format": "WAVE",
      "tracks": [
        {
          "artist": "Morelasoul",
          "duration": null,
          "pos": 3,
          "start": {
            "minutes-seconds-frames": "00:00:00",
            "seconds": 0
          },
          "title": "L'Afrique"
        }
      ]
    },
    {
      "filename": "disc01.04.mp3",
      "format": "WAVE",
      "tracks": [
        {
          "artist": "Soulroots",
          "duration": null,
          "pos": 4,
          "start": {
            "minutes-seconds-frames": "00:00:00",
            "seconds": 0
          },
          "title": "L'Afrique"
        }
      ]
    },
    {
      "filename": "disc01.05.mp3",
      "format": "WAVE",
      "tracks": [
        {
          "artist": "R-Jay",
          "duration": null,
          "pos": 5,
          "start": {
            "minutes-seconds-frames": "00:00:00",
            "seconds": 0
          },
          "title": "L'Afrique"
        }
      ]
    },
    {
      "filename": "disc01.06.mp3",
      "format": "WAVE",
      "tracks": [
        {
          "artist": "Rico Rosa",
          "duration": null,
          "pos": 6,
          "start": {
            "minutes-seconds-frames": "00:00:00",
            "seconds": 0
          },
          "title": "L'Afrique"
        }
      ]
    },
    {
      "filename": "disc01.07.mp3",
      "format": "WAVE",
      "tracks": [
        {
          "artist": "Pascal Morais",
          "duration": null,
          "pos": 7,
          "start": {
            "minutes-seconds-frames": "00:00:00",
            "seconds": 0
          },
          "title": "L'Afrique"
        }
      ]
    },
    {
      "filename": "disc01.08.mp3",
      "format": "WAVE",
      "tracks": [
        {
          "artist": "Rancido",
          "duration": null,
          "pos": 8,
          "start": {
            "minutes-seconds-frames": "00:00:00",
            "seconds": 0
          },
          "title": "L'Afrique"
        }
      ]
    }
  ],
  "mbid": "12d5a9d5-aa95-4f25-aca9-e482b616a3ca",
  "title": "Mysteryland NL 2014: L'Afrique"
}